pages/ksphere/kommander/1.1.0-beta/security/distributed-authnz/index.md

```diff
@@ -32,9 +32,13 @@ There is no centralized authorization component in Kommander. Each component and
 
 Kommander provides an [interface to federate authorization RBAC rules](../../operations/access-control/) across multiple clusters.
 
-* The Kubernetes API server is using [RBAC Authorization](https://kubernetes.io/docs/reference/access-authn-authz/rbac/).
+* The Kubernetes API server is using [RBAC Authorization][rbac_authz].
 
-* `traefik-forward-auth` applies [RBAC rules to authorize](/ksphere/konvoy/latest/security/external-idps/rbac/#portal-authorization) access based on an HTTP request. The [default Kommander workspace](../../workspaces/) comes with preconfigured Roles for accessing different operational portal services with View, Edit or Admin permissions.
+* `traefik-forward-auth` applies [RBAC rules to authorize](../../../../konvoy/1.5/security/external-idps/rbac/#portal-authorization) access based on an HTTP request. The [default Kommander workspace](../../workspaces/) comes with preconfigured Roles for accessing different operational portal services with View, Edit, or Admin permissions.
```

Comment:
the full path `/ksphere/konvoy/latest` seems better than a relative path here specific to 1.5. Did the original link not work? Also, are the instructions only valid for 1.5? 

Lastly, for sake of consistency can you follow link format `[][]` like you've done with the rest of the page?

Reply:
The one question I had, that I've seen eslewhere in this repo - if it's an internal link, it uses `[text](link)`
But, if it's external we do the `[text][link]`

Should we just make a decision for it on the page itself? Just try to be consistent there?

Reply:
my vote is for consistency. having all the links located in one place is easier to manage then some links at the bottom and some links scattered throughout the page

Reply:
Done, thanks @samvantran